tests/app-pip/app/main.py

```diff
@@ -0,0 +1,2 @@
+import urllib3
+print("Installed urllib3 version {}".format(urllib3.__version__))
```

Comment:
Maybe add the .pre-commit.yaml from the kick-start-python repository: https://github.com/godatadriven/kick-start-python/blob/master/.pre-commit-config.yaml#L1

Maybe not everything is relevant, but it will check on the line-ends.

Reply:
Added!